pandas/core/groupby/groupby.py

```diff
@@ -2784,10 +2784,11 @@ def cummin(self, axis=0, **kwargs):
         -------
         Series or DataFrame
         """
+        skipna = kwargs.get("skipna", True)
```

Comment:
can this go in the signature explicitly?

Reply:
Some context in https://github.com/pandas-dev/pandas/pull/41854#discussion_r646716955.

Happy to make that change here, was planning on doing in followup to make `skipna`, `numeric_only` consistent for `cumsum`, `cummin/max`

Reply:
yeah i agree we *want* this in the signature, can be a followup is ok.